craft_application/models/spread.py

```diff
@@ -0,0 +1,242 @@
+# This file is part of craft-application.
+#
+# Copyright 2024 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License version 3, as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+# SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Models representing spread projects."""
+
+
+from typing_extensions import Any, Self
+
+from craft_application.models import CraftBaseModel
+
+# Simplified spread configuration
+
+
+class SimpleSpreadSystem(CraftBaseModel):
```

Comment:
Simple is a bit misleading as at least for me subconsciously it would indicate a subset, not one of the inputs that is produced to generate a final `spread.yaml` 
We should come with a better naming like `UserDefined` or `Input`, as we'll probably at some point add also a validator for that.
```suggestion
class UserSpreadSystem(CraftBaseModel):
```

Reply:
Names will be replaced.